scania-digital-design-system/sdds
# feat(footer): migrate footer component

**Describe pull-request**  


**Solving issue**  
Fixes: [AB#2389](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2389)

**How to test**  
_Add description how to test if possible_
1. Go to storybook link below
2. Check in components -> footer
3. Check that the design is aligned with the one in Figma. 

**Screenshots**  
-

**Additional context**  
-
